### ISSUE TITLE
Uncaught TypeError: log.warning is not a function

### ISSUE THREAD

**Issue by drango** [pmarinov/rrss#8]
source file: rrss/rtable_gdrive.js:366:25
stack: 
```
TypeError: log.warning is not a function
    at rrss/rtable_gdrive.js:366:25
    at https://apis.google.com/_/scs/apps-static/_/js/k=oz.gapi.en.PbBqUN5CsYY.O/m=auth,client/exm=drive_realtime,drive_share/rt=j/sv=1/d=1/ed=1/am=AQ/rs=AGLTcCPmVtknW0nGp1uQh0Fe0NYz0LV71w/cb=gapi.loaded_1:447:173
    at https://apis.google.com/_/scs/apps-static/_/js/k=oz.gapi.en.PbBqUN5CsYY.O/m=auth,client/exm=drive_realtime,drive_share/rt=j/sv=1/d=1/ed=1/am=AQ/rs=AGLTcCPmVtknW0nGp1uQh0Fe0NYz0LV71w/cb=gapi.loaded_1:415:226
    at Object. (https://apis.google.com/_/scs/apps-static/_/js/k=oz.gapi.en.PbBqUN5CsYY.O/m=auth,client/exm=drive_realtime,drive_share/rt=j/sv=1/d=1/ed=1/am=AQ/rs=AGLTcCPmVtknW0nGp1uQh0Fe0NYz0LV71w/cb=gapi.loaded_1:346:475)
    at Object._.T.E.__cb (https://apis.google.com/_/scs/apps-static/_/js/k=oz.gapi.en.PbBqUN5CsYY.O/m=auth,client/exm=drive_realtime,drive_share/rt=j/sv=1/d=1/ed=1/am=AQ/rs=AGLTcCPmVtknW0nGp1uQh0Fe0NYz0LV71w/cb=gapi.loaded_1:35:262)
    at g (https://apis.google.com/_/scs/apps-static/_/js/k=oz.gapi.en.PbBqUN5CsYY.O/m=auth,client/exm=drive_realtime,drive_share/rt=j/sv=1/d=1/ed=1/am=AQ/rs=AGLTcCPmVtknW0nGp1uQh0Fe0NYz0LV71w/cb=gapi.loaded_1:29:62)
    at c (https://apis.google.com/_/scs/apps-static/_/js/k=oz.gapi.en.PbBqUN5CsYY.O/m=auth,client/exm=drive_realtime,drive_share/rt=j/sv=1/d=1/ed=1/am=AQ/rs=AGLTcCPmVtknW0nGp1uQh0Fe0NYz0LV71w/cb=gapi.loaded_1:20:26)
```

**Comment by pmarinov** [comment]
Thank you for reporting this.

It was a typo error of mine, the correct function is `log.warn()`. I will fix that in the next release.

But the warning is a valid concern. In your Google Drive, if you go to folder "Apps/rrss" what do you see listed there?

**Comment by pmarinov** [comment]
I was unable to find-out why exactly this case is hit.  I implemented a work-around which I've used for a month, it should be in version 0.8.0.

The problem is that, unclear why to me, a second table file is created on GDrive, then that one is used instead the original table. All data is, as a result of this, lost.

The work-around is to always use the oldest file. Side effect is that the multiple files will persist on GDrive.
